ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -6127,7 +6129,9 @@ void HqlCppTranslator::doBuildCall(BuildCtx & ctx, const CHqlBoundTarget * tgt,
         const char * name = str(external->queryId());
         if (getStringValue(nameTemp, queryAttributeChild(external, timerAtom, 0)).length())
             name = nameTemp;
-        buildHelperTimer(ctx, boundTimer, name);
+        // Grab optional ThorStatOption enum (as an int)
+        int statOption = getIntValue(queryAttributeChild(external, timerAtom, 1), 0);
```

Comment:
type should be consistent with the call above (int v __int64)

Reply:
Fixed.